sympy/core/tests/test_subs.py

```diff
@@ -640,7 +640,7 @@ def test_simultaneous_subs():
     assert (x/y).subs(reps) != (y/x).subs(reps)
     assert (x/y).subs(reps, simultaneous=True) == \
         (y/x).subs(reps, simultaneous=True)
-    assert Derivative(x, y, z).subs(reps, simultaneous=True) == \
```

Comment:
This change seems unrelated to this PR.

Reply:
Yeah sorry, I didn't remove it when I removed the other changes.
I will remove it as it is included in the other PR.

Reply:
Done